easybuild/easyblocks/m/mxnet.py

```diff
@@ -63,11 +63,15 @@ class EB_MXNet(MakeCp):
     """Easyblock to build and install MXNet"""
 
     @staticmethod
-    def extra_options(extra_vars=None):
+    def extra_options():
         """Change default values of options"""
-        extra = MakeCp.extra_options()
+        extra_vars = {
+            'install_r_ext': [False, "Enable installation of R extensions", CUSTOM],
```

Comment:
If we set the default to `None`, we can keep this backwards compatible by only auto-enabling it for old versions of MXNet:

```python
        extra_vars = {
            'install_r_ext': [None, "Enable installation of R extensions", CUSTOM],
}
```

and then in the `__init__` below:

```python
# auto-enable building of R extensions only for old versions of MXNet (< 1.0),
# since for newer versions is broken
if self.cfg['install_r_ext'] is None:
    if LooseVersion(self.version) < LooseVersion('1.0'):
      self.cfg['install_r_ext'] = True
    else:
      self.cfg['install_r_ext'] = False
```